lib/crawler/keyword/queries/keyword_query.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Crawler.Keyword.Queries.KeywordQuery do
+  import Ecto.Query
+
+  alias Crawler.Keyword.Schemas.Keyword
+
+  def user_keywords_query(user_id) do
+    Keyword
+    |> where([k], k.user_id == ^user_id)
```

Comment:
```suggestion
    |> where(user_id == ^user_id)
```

Reply:
Could you check this [reply](https://github.com/vnntsu/elixir-ic/pull/36#discussion_r1057493903)?

Reply:
I see